app/views/storage_locations/_source.html.erb

```diff
@@ -7,6 +7,7 @@
 <%= source.association association_field,
   collection: storage_locations,
   label: label,
+  disabled: defined?(disabled) ? disabled : false,
```

Comment:
Why was this added?

Reply:
I am trying to use same `_form` partial for both `edit` and `new` page. But in `edit` page the storage location should not be changed. That's why I did this.

Reply:
ok